Source/Core/Common/OpenXR.h

```diff
@@ -0,0 +1,120 @@
+// Copyright 2019 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <openxr/openxr.h>
+
+#include <atomic>
+#include <memory>
+#include <string>
+#include <thread>
+#include <vector>
+
+#include "Common/MathUtil.h"
+#include "Common/Matrix.h"
+
+namespace OpenXR
+{
+bool IsInit();
```

Comment:
```suggestion
bool IsInitialized();
```

There's no need to mince words here :p